sycl/source/detail/stream_impl.cpp

```diff
@@ -76,7 +99,7 @@ void stream_impl::flush() {
                 cgh, range<1>(BufferSize_), id<1>(OffsetSize));
     // Create accessor to the flush buffer even if not using it yet. Otherwise
     // kernel will be a leaf for the flush buffer and scheduler will not be able
-    // to cleanup the kernel. TODO: git rid of finalize method by using host
+    // to cleanup the kernel. TODO: get rid of finalize method by using host
```

Comment:
Is this TODO still relevant?

Reply:
Yes, this is still relevant unfortunatelly.